run-elrond-tests.py

```diff
@@ -344,14 +375,23 @@ def get_contract_code(code, filename):
     raise Exception('Currently only support getting code from file, or empty code.')
 
 def get_steps_sc_deploy(step, filename, output_dir):
+    k_steps = []
     tx = mandos_to_deploy_tx(step['tx'], filename, output_dir)
-    expect = mandos_to_expect(step['expect'])
-    return [KApply('scDeploy', [tx, expect])]
+    k_steps.append(tx)
+    if 'expect' in step:
+        expect = mandos_to_expect(step['expect'])
+        k_steps += expect
+    return k_steps
 
 def get_steps_sc_call(step, filename):
+    k_steps = []
     tx = mandos_to_call_tx(step['tx'], filename)
-    expect = mandos_to_expect(step['expect'])
-    return [KApply('scCall', [tx, expect])]
+    k_steps.append(tx)
+    expect = KApply('.Expect', [])
```

Comment:
No. Previously `scCall` has two arguments `transaction` and the `expect`. Now we separate these two since Expect is optional and there are many fields inside Expect are also optional.

Reply:
`scDeploy` is updated similarly